mysql-test/suite/innodb/t/percona_bug_1634932.test

```diff
@@ -0,0 +1,50 @@
+--echo #
+--echo # Bug 1634932: Assertion failure in thread x in
+--echo # file fts0que.cc
+--echo #
+
+-- source include/have_innodb.inc
+
+# Must have debug code to use SET SESSION debug
+--source include/have_debug.inc
+
+create table `t1` (
+	`FTS_DOC_ID` BIGINT unsigned NOT NULL AUTO_INCREMENT,
+	`text_content` mediumtext, PRIMARY KEY (`FTS_DOC_ID`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+CREATE UNIQUE INDEX FTS_DOC_ID_INDEX on t1(FTS_DOC_ID);
+
+SET autocommit=0;
+
+delimiter //;
+create procedure populate_t1()
+begin
+        declare i int default 1;
+        while (i <= 250) DO
+          insert into t1 (text_content) values ("some_text_1234 aaa");
+          set i = i + 1;
+        end while;
+end//
+
+delimiter ;//
+
+call populate_t1;
+SET autocommit=1;
+
+let @saved_session_debug= @@session.debug;
+SET SESSION debug="d,fts_instrument_result_cache_limit";
```

Comment:
But here you want to enable some debug setting and later disable it, so for that here use "+d,fts_...", and below "-d,fts_..."